driver/longhorn

```diff
@@ -224,7 +229,7 @@ formatDevIfNeed() {
     if OUT=$(blkid ${fullDevName}); then
         oldFsType=$(echo ${OUT} | grep -E -o 'TYPE=\"[[:alnum:]]{1,}\"' | awk -F"\"" '{print $2}')
         debug "dev ${fullDevName} have already fstype ${oldFsType}"
-        if [ "${oldFsType}" != "${newFsType}" ]; then
+        if [ "${oldFsType}" != "${newFsType}" ] && [ "${baseImage}" == "null" ]; then
```

Comment:
I think we should still reject the new FS if it's different from the existing fs even if base image is set?